### DISCUSSION TITLE
Naming npm tokens

### DISCUSSION THREAD

**Discussion by ghost** [npm/feedback#67] [Website] [answered]
Right now when tokens are created, we can't set a name to it. Sometimes we create more than one token. It is hard to remember them. It would be so nice if we could name it. Like the `note` while creating PAT in Github.

**Comment by ethomson** [comment]
Yes, I agree.  Definitely when I was working through some of the access token functionality and testing it, I was frustrated by this myself.  😁

**Comment by MylesBorins** [comment, marked as answer]
We've created a work item for this on the roadmap. Thanks for suggesting @sibiraj-sr 

https://github.com/npm/roadmap/issues/21
